step-controller/step-controller-server/src/main/java/step/core/deployment/AbstractStepServices.java

```diff
@@ -72,4 +74,16 @@ protected ObjectEnricher getObjectEnricher() {
 	protected ObjectFilter getObjectFilter() {
 		return objectHookRegistry.getObjectFilter(getSession());
 	}
+
+	protected AuthorizationManager<User, Session<User>> getAuthorizationManager(){
+		return getContext().get(AuthorizationManager.class);

```

Comment:
If the AuthorizationManager is required, it is recommended to use the methode getContext().require